lib/wicked_pdf/wicked_pdf_helper/assets.rb

```diff
@@ -27,6 +27,21 @@ def wicked_pdf_stylesheet_link_tag(*sources)
         end.html_safe
       end
 
+      def wicked_pdf_stylesheet_pack_tag(*sources)
+        return unless defined?(Webpacker)
+
+        if Webpacker.dev_server.running?
+          stylesheet_pack_tag(*sources)
+        else
+          css_text = sources.collect do |source|
+            source = WickedPdfHelper.add_extension(source, 'css')
+            source_path = asset_pack_path(source)
+            wicked_pdf_stylesheet_link_tag(root_url[0...-1] + source_path)
```

Comment:
It occurs error when request spec.
I think that is better in the following code.

```ruby
wicked_pdf_stylesheet_link_tag(Rails.public_path.join(source_path))
```

Reply:
I believe this is a riskier solution, since it uses filesystem instead of URLs. I believe that direct links to filesystem should be avoided as much as possible